go.mod

```diff
@@ -2,6 +2,7 @@ module github.com/edgexfoundry/go-mod-messaging
 
 require (
 	github.com/eclipse/paho.mqtt.golang v1.2.0
+	github.com/go-redis/redis/v7 v7.2.0
```

Comment:
For post Geneva tech debt, I believe we should decide on go-redis or redigo and use one library throughout.

Reply:
Interestingly I added `github.com/gomodule/redigo` when I first started and my IDE (GoLand) changed it to this. I tried this again with `github.com/gomodule/redigo HEAD` which should grab the latest tagged version and it changed to `go-redis`. Sorry, I did not catch this earlier.